lib/internal/Magento/Framework/Mview/View/ChangelogInterface.php

```diff
@@ -11,6 +11,11 @@
  */
 interface ChangelogInterface
 {
+    const ATTRIBUTE_SCOPE_SUPPORT = 'attribute_scope';
+    const STORE_SCOPE_SUPPORT = 'store_scope';
+    const ATTRIBUTE_COLUMN = 'attribute_id';
```

Comment:
Should be defined in eav module?

Reply:
Also visibility should be provided whether it's reusable const and a part of API (`public`) or part of the implementation (`protected` / `private`).